src/main/scala/Dao/DAO.scala

```diff
@@ -1,21 +1,20 @@
 package Dao
 
-import java.util.UUID
-import scala.collection.mutable.ListBuffer
 import models.Users
+
 import scala.concurrent.Future
 
 trait DAO {
 
   def addUser(user: Users): Future[String]
 
-  def getById(userId: UUID): Future[Option[Users]]
+  def getById(userId: Int):   Future[List[Users]]
```

Comment:
This should be an `Future[Option[User]]` as you are returning only a single user.